src/mmw/js/src/core/views.js

```diff
@@ -673,6 +679,24 @@ var MapView = Marionette.ItemView.extend({
             }
         });
     },
+
+    renderFocusBounds: function() {
+        var bbox = this.model.get('focusBounds');
+
+        this._focusBoundsLayer.clearLayers();
+
+        if (bbox) {
+            var xmin = bbox[0],
+                ymin = bbox[1],
+                xmax = bbox[2],
+                ymax = bbox[3],
+                bounds = [[ymin, xmin], [ymax, xmax]],
+                rect = new L.Rectangle(bounds);
+
+            this._focusBoundsLayer.addLayer(rect);
+            this._leafletMap.fitBounds(bounds);
+        }
+    }
```

Comment:
The primary purpose of including this feature is mainly for troubleshooting search result boundaries. We'll need to extend this or rewrite this to support showing multiple boxes/points. See #1940.